api_docs/server_doc.py

```diff
@@ -0,0 +1,350 @@
+                    "@type": "Operation",
+                    "expects": "http://schema.org/Action",
+                    "method": "POST",
+                    "possibleStatus": [
+                        {
+                            "code": 200,
+                            "description": "Successfully charged"
+                        }
+                    ],
+                    "returns": "http://schema.org/fuelCapacity",
+                    "title": "GetTemperature"
+                }
+            ],
+            "supportedProperty": [
+                {
+                    "@type": "SupportedProperty",
+                    "property": "http://auto.schema.org/speed",
+                    "readable": "true",
+                    "required": "true",
+                    "title": "Speed",
+                    "writeable": "true"
+                },
+                {
+                    "@type": "SupportedProperty",
+                    "property": "http://schema.org/geo",
+                    "readable": "true",
+                    "required": "true",
+                    "title": "Position",
+                    "writeable": "true"
+                },
+                {
+                    "@type": "SupportedProperty",
+                    "property": "http://schema.org/fuelCapacity",
+                    "readable": "true",
+                    "required": "true",
+                    "title": "Battery",
+                    "writeable": "true"
+                },
+                {
+                    "@type": "SupportedProperty",
+                    "property": "http://schema.org/device",
+                    "readable": "true",
+                    "required": "true",
+                    "title": "Sensor",
+                    "writeable": "false"
+                },
+                {
+                    "@type": "SupportedProperty",
+                    "property": "http://schema.org/model",
+                    "readable": "true",
+                    "required": "true",
+                    "title": "Model",
+                    "writeable": "false"
+                },
+                {
+                    "@type": "SupportedProperty",
+                    "property": "https://schema.org/status",
+                    "readable": "true",
+                    "required": "true",
+                    "title": "Status",
+                    "writeable": "true"
+                }
+            ],
+            "title": "Drone"
+        },
+        {
+            "@context": "http://hydrus.com/context/LogEntry.jsonld",
+            "@id": "http://hydrus.com/serverapi/vocab/LogEntry",
+            "@type": "Class",
+            "description": "Class for a log entry",
+            "supportedOperation": [
+                {
+                    "@id": "/serverapi/log_entry",
+                    "@type": "Operation",
+                    "expects": "http://hydrus.com/LogEntry",
+                    "method": "POST",
```

Comment:
I don't think we need post requests for log_entry, log entries are automatically generated from various drone datastream and status submissions. I think it should be get.